akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphFlexiMergeSpec.scala

```diff
@@ -546,16 +546,16 @@ class GraphFlexiMergeSpec extends AkkaSpec {
       s.expectNext("onInput: e")
       s.expectNext("onInput: c")
       s.expectNext("onInput: f")
-      s.expectNext("onComplete: 2")
```

Comment:
the expected string `"onUpstreamFinish` is generated by the TestMerge at `CompletionHandling.onUpstreamFinish`, so for me it makes sense to name it like that.